minimatch.js

```diff
@@ -708,11 +698,6 @@ function match (f, partial) {
 
   var options = this.options
 
-  // windows: need to use /, not \
-  if (path.sep !== '/') {
```

Comment:
It's wrong. On Windows '/' character in pattern should match both '/' and '\' characters in file paths. There is not ideal slow implementation (i think this should be done in matching engine) but it work.